src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,18 @@
 package core.basesyntax;
 
+import java.util.function.Supplier;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
  * выбора цвета</p>
  */
-public class RandomColor {
+public class RandomColor implements Supplier<String> {
+
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+
+    @Override
+    public String get() {
+        return COLORS[(int) (Math.random() * 7)];
```

Comment:
don't use magic numbers. And try to use new Randon().nextInt()

Reply:
done